src/coverlet.msbuild.tasks/CoverageResultTask.cs

```diff
@@ -74,16 +77,18 @@ public string CoverletMultiTargetFrameworksCurrentTFM
 
         public CoverageResultTask()
         {
-            _logger = new MSBuildLogger(Log);
+           _serviceProvider = new Services().GetServiceProvider(Log);
```

Comment:
move this to `Execute` so we're aligned with other VSTask

Reply:
Done